packages/components/psammead-media-player/src/index.test.jsx

```diff
@@ -86,4 +86,20 @@ describe('Media Player: Canonical Entry', () => {
       title="Audio player"
     />,
   );
+
+  shouldMatchSnapshot(
+    'renders a placeholder image with guidance',
+    <CanonicalMediaPlayer
+      placeholderSrc="http://foo.bar/placeholder.png"
+      src="http://foo.bar/iframe"
+      service="news"
+      mediaInfo={{
+        title: 'Dog chases cat.',
+        guidance:
```

Comment:
Why is this called `guidance` here but [above](https://github.com/bbc/psammead/pull/2517/files#diff-57d225e3260d5fe9f558b44c21f90476) is `guidanceMessage`?